app/templates/editor/common/edit.pug

```diff
@@ -0,0 +1,29 @@
+extends /templates/base
+
+block content
+  div
+    ol.breadcrumb
+      li
+        a(href="/editor", data-i18n="editor.main_title") CodeCombat Editors
+      li
+        a(href=`/editor/${view.resourceName}`, data-i18n="editor.resource_title")
```

Comment:
This says "Resource Editor", specific to ResourceHubResources. Need new i18n strings, "editor.podcast_title" and "editor.podcast_edit_title".